docs/rfcs/rfc_0001_deductions_from_the_mission_statement.md

```diff
@@ -0,0 +1,111 @@
+# Deductions from the mission statement
+
+by: Crazy Techno Opossum
+
+This document is a Request For Comments. It is open for criticism and review.
+We expect review to improve the quality of this document.
+
+This document extracts some deductions and technical constraints from the mission statement.
+This document does not define a system that maps those constraints to a particular implementation.
+
+This document's intended audience is people that want to help build the system.
+Some cryptography and mathematical knowledge is expected.
+
+
+## Summary
+
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+
+## Key points from the mission statement
+
+> Imagine a unique social project that takes a web server and turns it into a multiplayer hub for 
+> a purposeful cause. The goal? To create a space where people come together, enjoy games or 
+> activities, and contribute to a shared fund. 
+> This fund isn’t just any fund – it’s a resource that automatically gets divided among the users of the server. 
+> So, the more you engage and participate, the more you receive from the fund. 
+> This innovative project not only brings people closer through fun and interaction but also 
+> enables them to collectively impact their community by distributing funds where they’re needed most.
+
+Technology wise, we extract the following points:
+- A web server.
+- Games and activities.
+- People can contribute to a shared fund.
+- The fund gets distributed among the users of the server.
+- A user's income from the fund is a function of their engagement.
```

Comment:
There are multiple options, 
Distribute to everyone
randomly
based on engagement

we distribute a base level sharing mechanism and let the open source community test these options in practice.

Reply:
We still need to define alternatives and implement them for them to be tested.
So we need to come up with the specific options.

Reply:
yes,i would do that as additional modules, like a plugin store

everyone is what we have
random is just using rand range
engagement is more complicated , bc we basically have a open user endpoint,like a api